client/src/components/GraphViewerComponent/GraphViewerComponent.js

```diff
@@ -891,10 +892,16 @@ class GraphViewerComponent extends React.Component {
               <div className="alert--message">
                 {noResults ? <span>No results found. </span>
                   : <>
-                    {relationshipsOnly ? <span>You can only render relationships if a twin is returned too. </span>
+                    {relationshipsOnly
+                      ? <><span>A graph may only be rendered if the results contain a twin. </span>
+                        <span>Click here to open the</span>
```

Comment:
@JoseZarmada - i think we need to add a space after "the"

![image](https://user-images.githubusercontent.com/13650672/156078694-0e9d7e25-41be-46f3-82b3-8e70df936e27.png)